streams/src/test/java/org/apache/kafka/streams/state/internals/StreamThreadStateStoreProviderTest.java

```diff
@@ -236,5 +238,10 @@ public Sensor addLatencySensor(final String scopeName,
         public void recordLatency(final Sensor sensor, final long startNs, final long endNs) {
 
         }
+
+        @Override
+        public Map<MetricName, ? extends Metric> metrics() {
+            return null;
```

Comment:
As above. Maybe we need to have a `NoOpStreamsMetrics` class that can be used for tests?